tests/expected-output-mig-single.txt

```diff
@@ -30,4 +30,5 @@ nvidia\.com\/gpu\.engines\.jpeg=[0-9]+
 nvidia\.com\/gpu\.engines\.ofa=[0-9]+
 nvidia\.com\/gpu\.slices\.gi=[0-9]+
 nvidia\.com\/gpu\.slices\.ci=[0-9]+
+nvidia\.com\/gpu\.mode=[unknown|compute|graphics]
```

Comment:
nit: this can only be `compute` now, correct? Not a blocker.

Reply:
Fixed